res/values/strings.xml

```diff
@@ -28,6 +28,7 @@
     <string name="drawer_open">Open</string>
     <string name="prefs_category_general">General</string>
     <string name="prefs_category_more">More</string>
+    <string name="prefs_category_security"> Security </string>
```

Comment:
You do not need the extra white spaces at the beginning and end in these string resources.